s3_website.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files`
                       .split("\n")
                       .reject { |f| f.match('sbt-launch.jar') } # Reject the SBT jar, as it is a big file
-                      .push('resources/s3_website.jar.md5') # Include the checksum file in the gem
```

Comment:
Thanks for the explanation, @pjkelly. I have never ran into that problem because I've always had the *s3_website.jar.md* there. Clearly this is not the case for devs like you who fork the repo and start from scratch.